package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "main": "dist/src/core.js",
   "dependencies": {
-    "@decentralized-identity/sidetree": "0.10.0",
```

Comment:
Since we're using an unofficial release of sidetree, would it be better if we merge into dev instead of master?